lib/net/rpc.go

```diff
@@ -0,0 +1,38 @@
+package net
+
+import (
+	"bufio"
+	"errors"
+	"io"
+	gonet "net"
+	"net/http"
+	"net/rpc"
+)
+
+func dialHTTPPath(dialer Dialer, network, address, path string) (
+	*rpc.Client, error) {
+	var err error
```

Comment:
You may want to consider letting the caller pass an empty string and default to `rpc.DefaultRPCPath`